Assets/Scripts/Gamestate/MatchController.cs

```diff
@@ -0,0 +1,141 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine.SceneManagement;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+/// <summary>
+/// Wrapper struct for tying refference to Player class with the in-game player.
+/// </summary>
+public struct MatchPlayer
+{
+    public MatchPlayer(Player player, PlayerStateController playerStateController, int startAmount)
+    {
+        this.player = player;
+        playerStateController.chips = startAmount;
+        this.playerStateController = playerStateController;
+    }
+    // Reference to player identity class
+    public Player player;
+    // Reference to in-match player
+    public PlayerStateController playerStateController;
```

Comment:
The comment and my  _professional_ opinion on variable and type naming make begs the question if `PlayerStateController` cannot be renamed to something more explicit than `Controller`

Reply:
Agreed, though I'm not sure what we should rename it to. Simply `PlayerState`?

Reply:
What we want to describe with the name is "All information a single player holds exclusive to a single match", of which I had hard time to create a good name from. Very open for suggestions 😅

Reply:
As per discussed in meeting, class is renamed to simply `Player`

Reply:
No, in fact it is renamed to `PlayerManager`, `Player` is renamed to `PlayerIdentity` and `MatchPlayer` is renamed to `Player`. ~~This makes sense, I swear.~~